app/models/manageiq/providers/ansible_tower/automation_manager.rb

```diff
@@ -8,14 +8,20 @@ class << self
   end
 
   delegate :authentications,
+           :authentications=,
            :authentication_check,
            :authentication_status,
            :authentication_status_ok?,
            :connect,
+           :endpoints,
+           :endpoints=,
+           :url,
+           :url=,
            :verify_credentials,
            :with_provider_connection,
            :to => :provider
 
+  belongs_to :provider, :autosave => true, :dependent => :destroy
```

Comment:
Was or is there a `belongs_to :provider` somewhere else? Trying to understand how the existing delegations were working.

Reply:
Yes there is in core but we need the autosave and dependent destroy